app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -91,6 +101,8 @@ public void showProgressBar(boolean show) {
 
     @Override
     public void onLoginSuccess() {
+        if(etEmail.getEditText().getText() != null)
+            presenter.saveEmail(etEmail.getEditText().getText().toString());
```

Comment:
Similarly here, take this action in presenter in the login callback, if it is successful. We add all logic to presenter so that we can unit test it